wherehows-web/app/constants/data/temp-mock/health.ts

```diff
@@ -0,0 +1,48 @@
+import { IHealthScore } from 'wherehows-web/typings/api/datasets/health';
```

Comment:
Why not have this in mirage? @cptran777

Reply:
@theseyi It is/was in mirage, but since mirage doesn't exist in staging, we have a breaking import 'module doesn't exist' error

Reply:
That's true. on the flip side is there a need to push mock data as part of the commit? the component's can exist without a live data source, and then once the api is integrated they'll be reified. just a thought about throw away code

Reply:
but i guess this is ok since it's merged 👍